beginner.md

```diff
@@ -0,0 +1,588 @@
+# delite - creating custom components
+
+## delite background
+`delite` is a new JavaScript library born out of the [Dojo Toolkit Dijit framework](http://dojotoolkit.org/reference-guide/1.10/dijit).
```

Comment:
You could think of it as a strategic decision (and I think it's a good strategy), or you could just remove it because (as cjolif mentioned) it's not relevant to the tutorial.

Maybe someday we'll have a tutorial specifically for migrating dijit widgets.  Not sure if that would be useful since we already have a migration guide.